Removing Deprecated Function

PropelledDog opened the pull request:
A summary of my change(s):
- `ID2D1Factory::GetDesktopDpi()` is deprecated and the Visual Studio compiler will no longer compile new code using this function.
    - The compiler suggests using the [`GetDpiForWindow()`](https://docs.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-getdpiforwindow) function for desktop applications.

The following has been tested on Windows 11 using Visual Studio Community 2022 and shown to be working correctly.

PropelledDog committed:
Removing deprecated function

A summary of my change(s):
- `ID2D1Factory::GetDesktopDpi()` is deprecated and the Visual Studio compiler will no longer compile code using this function.

PRMerger16 commented:
@PropelledDog : Thanks for your contribution! The author(s) have been notified to review your proposed change.

PropelledDog committed:
Fixing Ambiguous Grammar

Removing the ambiguous subject `it` from comment as this was no longer grammatically correct following my updated code change.

PRMerger9 commented:
@PropelledDog : Thanks for your contribution! The author(s) have been notified to review your proposed change.

stevewhims commented:
Thank you, I'm investigating. :)

-Steve

PropelledDog commented:
Hey @stevewhims, any update on your investigation? I can provide more evidence of this function deprecation, if desired! :)

-Dan

stevewhims commented:
Thanks, I've pinged the internal thread again.

-Steve

PropelledDog commented:
Hey @stevewhims, were you able to get into contact with the internal thread team? Is there anything I can do to help the internal thread team here?

Apologizes for the bump again - I just wanted to check in to hear if there are any issues with the above change. If there is anything I can do to help add more confidence here, then definitely let me know!

stevewhims commented:
UPDATE: Thanks for this PR! Yes, the function is deprecated, as you say. I fixed the code in this topic (and in many others) in my own PRs since there were quite a few topics using this old API. The technique we went with was slightly different from the one in this PR (you can see it in the live topic now). Thanks again! I'll go ahead and close this PR.

-Steve
